docs/pages/deploy-a-cluster/helm-deployments/custom.mdx

```diff
@@ -38,16 +38,21 @@ of this page to choose the correct version.
 
 ## Step 3/4. Setting up a Teleport cluster with Helm using a custom config
 
-In `scratch` mode, the `teleport-cluster` Helm chart generates a minimal
-configuration and lets you pass your custom configuration through the chart's values.
+`teleport-cluster` deploys two sets of pods: proxy and auth. You can provide two configurations, one for each pod type.
 
-`teleport-cluster` deploys two sets of pods: proxy and auth.
-You must provide two configurations, one for each pod type.
+- Any values set under the `proxy` section of your chart values will be applied to the proxy service pods
+  only. You can provide custom YAML under `proxy.teleportConfig` to override elements of the default Teleport
+  proxy service configuration with your own.
+- Any values set under the `auth` section of your chart values will be applied to the auth service pods only.
+  You can provide custom YAML under `auth.teleportConfig` to override elements of the default Teleport auth
+  service configuration with your own.
 
-- The `proxy` pod configuration should contain at least the `proxy_service`
-  section and the `teleport` section without the `storage` part.
-- The `auth` pod configuration should contain at least the `auth_service` and
-  `teleport` sections.
+Any YAML you provide under a `teleportConfig` section will be merged with the chart's default YAML configuration,
+with your overrides taking precedence. This allows you to override only the exact behaviour that you need, while keeping
+the Teleport-recommended chart defaults for everything else.
+
+Also, note that many useful Teleport features can already be configured using chart values rather than custom YAML.
+Setting `publicAddr` is the same as setting `proxy.teleportConfig.proxy_service.public_addr`, for example.
 
 The chart automatically creates a Kubernetes join token, named after the Helm
```

Comment:
If we don't go full `scratch` anymore, we could drop this paragraph about token creation as this should be transparent.

Reply:
I agree, in most cases people probably won't need to override the default join behaviour because Kubernetes "just works" here.